src/GameWorldSimulator/NeoServer.Game.World/Models/Tiles/StaticTile.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Text.RegularExpressions;
```

Comment:
is this being used?

Reply:
removed on last commit: https://github.com/OpenCoreMMO/OpenCoreMMO/pull/665/commits/82314937fa9eeedb50263b395791ec30a500404b